app/assets/stylesheets/partials/_overlay.scss

```diff
@@ -7,7 +7,7 @@
   height: 100%;
   width: 100%;
   z-index: 2;
-  background-color: rgba(63, 60, 60, 0.705);
+  background-color: rgba(3, 51, 45, 0.39);
```

Comment:
In general we should be setting colors as SCSS variables. See https://github.com/sds/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorvariable

Reply:
Done